Src/BootCamp.Chapter/Logging/ILog.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public interface ILog
+    {
+        public void LogOpenProgram();
```

Comment:
Logging should be a more generic operation. You just pass a single message and log it. You don't need specialized methods for logging open program, closing program or crash. However, it does make sense to have the following:
LogInfo
LogWarning
LogError

Please change the names, refactor.